google-cloud-spanner/src/test/resources/com/google/cloud/spanner/connection/ClientSideStatementsTest.sql

```diff
@@ -1931,9148 +1931,11005 @@ NEW_CONNECTION;
 @EXPECT EXCEPTION INVALID_ARGUMENT
 show variable/-read_only_staleness;
 NEW_CONNECTION;
-begin;
+show variable optimizer_version;
```

Comment:
I forgot to regenerate the test sql files for the `optimizer_version` feature, so that's why that is included here. The generator does not support generating tests for only some features, so it's impossible to keep that change separate from the changes for `CommitStats` without completely rewriting the generator.